includes/container/config.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
```

Comment:
Do we need a `namespace` for this file?

Reply:
@Batsirai-muchareva As discussed, we need to scope this to avoid naming collisions with other plugins that can use the same dependancies.

Reply:
Can you add some comment on this file to explain what it does?